src/config/topics.js

```diff
@@ -415,6 +415,7 @@ export const direktverbindungenIframe = {
   overlaySide: 'left',
 };
 
+const railplusPdfDownloadType = 'pdf-download';
```

Comment:
Move it to constants file, with proper casing.

Reply:
done